### ISSUE TITLE
Possible for a large mint to fail if gas prices change too much

### ISSUE THREAD

**Issue by glitchyg** [Montspy/LoopMintPy#25]
This tool worked great for minting all 1000 NFT's I had. The only issue i ran into was when the gas fee increased too much during my mint. This is the response that I got from the server:

```
{
	"resultInfo": {
		"code": 114002,
		"message": "Invalid feeAmount:734000000000000000 should be:925000000000000000"
	}
}
```

**Comment by Montspy** [comment]
Proposal for a partial fix/mitigation:
+ Add a fee slippage parameter (default: 50%)
~ Ask user to approve a range of fees for minting based on the fee slippage, rather than a single estimate
+ Detect code 114002 error and exit cleanly

**Comment by Montspy** [comment]
@glitchyg Draft PR mitigating the crash #28 

Example run (reproducing by choosing maxFeeAmount 10x lower than recommended by the API):
![image](https://user-images.githubusercontent.com/2715455/166392627-0408bcd3-4d5a-4ea2-b686-05ca6147d2c7.png)

**Comment by Montspy** [comment]
Need to verify that Loopring API accepts a maxFeeAmount greater than the recommended fee amount

**Comment by Montspy** [comment]
Fix should be applied to the transfer tool as well

**Comment by Montspy** [comment]
Added slippage to minter in b47daaf22dcbf7764418d46c390689fe8f8cadfc and verified that Loopring API accepts maxFeeAmount greater than the recommended fee amount
Fix applied to transfer tool in df951c8f6261dd0808fac971afa40a83cea77d58

**Comment by Montspy** [comment]
Merged #28 
Closing as fixed
